standalone_tests/sweep_check.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python
```

Comment:
nit: format file?

Reply:
also, a hint about how to run this / intended usage would be nice

Reply:
this is a copy from cli-og.   I will document it when it starts passing :)